core/components/atoms/dropdown/DropdownList.tsx

```diff
@@ -539,10 +539,30 @@ const DropdownList = (props: OptionsProps) => {
     );
   };
 
+  const GroupedListOptions = (listOptions: OptionSchema[]): OptionSchema[] => {
+    const groupList = listOptions.reduce((acc, option) => {
+      const group = option.group || '';
+
+      if (!acc[group]) {
+        acc[group] = []; // Create an empty array for the group if it doesn't exist
+      }
+
+      acc[group].push(option); // Push the option into the group's array
+
+      return acc;
+    }, {} as { [key: string]: OptionSchema[] }); // Type asserting {} to provide type of key and value
```

Comment:
Please remove the comments as it is very intuitive.

Reply:
Done